src/utils/create-ref-id.js

```diff
@@ -0,0 +1,4 @@
+export default () =>
+  Math.random()
```

Comment:
There is no way to seed the random-number generator in JS, so it might start always the same. How about multiplying with https://developer.mozilla.org/en-US/docs/Web/API/Performance/now?